docs/sql/statements/update.rst

```diff
@@ -28,6 +29,12 @@ UPDATE changes the values of the specified columns in all rows that satisfy the
 condition. Only the columns to be modified need be mentioned in the SET clause;
 columns not explicitly modified retain their previous values.
 
+The optional RETURNING clause causes UPDATE to compute and return value(s) based
+on each row actually updated. Any expression using the table's columns, and/or
+columns of other tables mentioned in FROM, can be computed. The new (post-update)
```

Comment:
We can only have a single table in the `FROM` clause. I think phrasing this like it is may be confusing because it is implying that you can have multiple relations in `FROM`.

Reply:
My bad, doing copy-paste here. After a closer look the update statement does actually not support `FROM` at all.